### PR TITLE
 New: Persistence layer for saving video data instead of in memory

### PR DESCRIPTION
**Pull request by jcarlosestela**

### CODE REVIEW

File: OCM/Data/Persistance/Video/Model/VideoDB.swift [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+//
+
+import Foundation
+import CoreData
+
+@objc(VideoDB)
+public class VideoDB: NSManagedObject {
+    
+    func toCachedVideo() -> CachedVideoData? {
+        guard let identifier = self.identifier, let url = self.url, let previewUrl = self.previewUrl, let type = self.type, let updatedAt = self.updatedAt as? Date, let format = VideoFormat.from(type) else { return nil }
```
Comment (udelunar) [inline OCM/Data/Persistance/Video/Model/VideoDB.swift]: I think allways we need put into guard a log.

File: OCM/Data/Persistance/Video/VideoPersister.swift [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    }
+    
+    func initDataBase() {
+        guard let managedObjectModel = self.managedObjectModel else { logWarn("managedObjectModel is nil"); return }
+        let coordinator = NSPersistentStoreCoordinator(managedObjectModel: managedObjectModel)
+        let url = self.applicationDocumentsDirectory.appendingPathComponent("VideoDB.sqlite")
+        do {
+            try coordinator.addPersistentStore(ofType: NSSQLiteStoreType, configurationName: nil, at: url, options: nil)
+        } catch let error {
+            print(error)
```
Comment (udelunar) [inline OCM/Data/Persistance/Video/VideoPersister.swift]: Please dont use print

**Review by udelunar — COMMENTED** [review]
Please revise comments

**Comment by jerilyngigigo** [comment]
LGTM

**Pull request merged**